quantum/split_common/transport.c

```diff
@@ -130,15 +131,20 @@ bool transport_master(matrix_row_t matrix[]) {
     matrix[i] = serial_s2m_buffer.smatrix[i];
   }
 
-#  if defined(RGBLIGHT_ENABLE) && defined(RGBLIGHT_SPLIT)
-  // Code to send RGB over serial goes here (not implemented yet)
-#  endif
-
 #  ifdef BACKLIGHT_ENABLE
   // Write backlight level for slave to read
   serial_m2s_buffer.backlight_level = backlight_config.enable ? backlight_config.level : 0;
 #  endif
 
+#  if defined(RGBLIGHT_ENABLE) && defined(RGBLED_SPLIT)
+  static rgblight_config_t prev_rgb = {~0};
+  uint32_t rgb = eeconfig_read_rgblight();
+  if (rgb != prev_rgb.raw) {
```

Comment:
Did you have an update for this that doesn't have the problem that we recently fixed for the i2c case for situations where the master to slave transfer fails?

Also, after merging in master, you should use `rgblight_read_dword()` instead of `eeconfig_read_rgblight()`

Reply:
Okay, I've updated the code to use `rgblight_read_dword()` instead of `eeconfig_read_rgblight() now.